packages/react-components/src/components/Picker/components/PickerList.tsx

```diff
@@ -32,6 +32,7 @@ export interface IPickerListProps {
     userProps?: React.HTMLProps<HTMLElement> | undefined
   ) => Record<string, unknown>;
   emptyStateText?: string;
+  hideWhenEmpty?: boolean;
```

Comment:
non-breaking API change: prop preventing the list from appearing when it's empty. I needed that for the AC component

Reply:
No need to show any empty state? 🤔